README.md

```diff
@@ -21,3 +21,19 @@ Scripts should be "chmod +x" executable in order to run as a cron job successful
 ## SES usage within alma-scripts
 * Emails from the SES service must come from `noreply@libraries.mit.edu` for this app 
 
+### Using Moto for local SSM testing
```

Comment:
I would label this section "for local development" as opposed to testing, and I would note below that developers *should* use moto for local development rather than develop against real SSM. In this case it's not desirable, it's a requirement.